### DISCUSSION TITLE
Welcome to SocketChat Discussions!

### DISCUSSION THREAD

**Discussion by profreelancer210** [profreelancer210/SocketChat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by profreelancer210** [comment]
I have responsive error.
Who can help me.

**Comment by profreelancer210** [comment]
I have frontend error
Who can help me.

**Comment by Luisa-m17** [comment]
It is about your width problem.
You will fix it as give percent width

**Reply by profreelancer210** [reply]
ok Thanks

**Reply by profreelancer210** [reply]
OK

**Comment by profreelancer210** [comment]
> It is about your width problem. You will fix it as give percent width
